.travis.yml

```diff
@@ -10,36 +10,13 @@ install:
     --username="$DOCKER_HUB_USERNAME" \
     --password="$DOCKER_HUB_PASSWORD"
 
+- |-
+  curl https://raw.githubusercontent.com/cloudposse/build-harness/master/scripts/travis_docker_tag.sh > travis_docker_tag.sh \
+  && chmod +x travis_docker_tag.sh
+
 script:
 - docker build --build-arg GITHUB_TOKEN=$GITHUB_TOKEN -t $DOCKER_IMAGE_NAME .
 
 after_success:
-- |-
-  if [ ! -z "$TRAVIS_TAG" ]; then \
-    docker tag $DOCKER_IMAGE_NAME  $DOCKER_IMAGE_NAME:$TRAVIS_TAG &&  \
-    docker push $DOCKER_IMAGE_NAME:$TRAVIS_TAG; \
-  fi
-
-- |-
-  if [ ! -z "$TRAVIS_PULL_REQUEST_BRANCH" ]; then \
-    docker tag $DOCKER_IMAGE_NAME $DOCKER_IMAGE_NAME:pr-$TRAVIS_PULL_REQUEST_BRANCH && \
-    docker push $DOCKER_IMAGE_NAME:pr-$TRAVIS_PULL_REQUEST_BRANCH; \
-  fi
+- ./travis_docker_tag.sh
```

Comment:
personal preference would be `make -C build-harness docker-tag` which calls travis since we're using travis